helm/afm/values.sample.yaml

```diff
@@ -1,5 +1,5 @@
 # Try it with `helm install --generate-name --dry-run -f helm/afm/values.sample.yaml helm/afm`
-labels:
+metadata:
```

Comment:
Please do leave the labels, you can add some examples of labels,
and remove the metadata at all (include name and namespace).

see example https://fybrik.io/dev/contribute/modules/#module-helm-chart.
P.S. the namespace label will be renamed.